### PR TITLE
feedback implemented, code clean halfway

### PR DESCRIPTION
**Pull request by chin2yere**

### CODE REVIEW

File: Petits-debuts-ui/src/Components/Trending/Trending.jsx [hunk 1]
```diff
@@ -24,9 +27,7 @@ export default function Trending({
   const { trending, setTrending } = useContext(TrendingContext);
   const { productContext } = useContext(ProductContext);
   const { TotalOther, setTotalOther } = useContext(TotalOtherContext);
-  //start trending
-
-  //this function updates the local storage for trending
+  const thresholdMinimumScore = 70;
```
Comment (zderick) [inline Petits-debuts-ui/src/Components/Trending/Trending.jsx]: Can we make `thresholdMinimumScore` capitalized?

File: Petits-debuts-ui/src/Components/Trending/Trending.jsx [hunk 2]
```diff
@@ -232,23 +228,29 @@ export default function Trending({
...
             //logic for exceptions
+            //typically each product is rated on a score out of 120
+            //but an absence of one or more pillar can reduce the total score to as low as 80
+            //we have to scale up to 120 regardless because our threshold mark is 70
             if (
               Object.keys(personalCart).length === 0 &&
               orderContext.length !== 0
             ) {
+              //this part scales up the score to 120 from 80
               score = (score * 120) / 95;
```
Comment (zderick) [inline Petits-debuts-ui/src/Components/Trending/Trending.jsx]: It's be great to define what 120, 95, 105 is. They are kind of arbirary

File: Petits-debuts-ui/src/Components/Trending/Trending.jsx (hunk 1 above)
Comment (chin2yere) [reply Petits-debuts-ui/src/Components/Trending/Trending.jsx]: i don't understand
you mean THRESHOLDMIMINUMSCORE?

File: Petits-debuts-ui/src/Components/Trending/Trending.jsx (hunk 2 above)
Comment (chin2yere) [reply Petits-debuts-ui/src/Components/Trending/Trending.jsx]: Those are the highest attatinable scores before scaling

i referenced their meaning a few lines above. do you want me to dig deeper?

File: Petits-debuts-ui/src/Components/Trending/Trending.jsx (hunk 1 above)
Comment (zderick) [reply Petits-debuts-ui/src/Components/Trending/Trending.jsx]: Something like THRESHOLD_MINIMUM_SCORE. This is a common pattern for constants. It's called snake case

Comment (chin2yere) [reply Petits-debuts-ui/src/Components/Trending/Trending.jsx]: okay

File: Petits-debuts-ui/src/Components/Trending/Trending.jsx (hunk 2 above)
Comment (zderick) [reply Petits-debuts-ui/src/Components/Trending/Trending.jsx]: Define them to be also constants of what they mean.

MAXIMUM_SCORE / WHATEVER_95_MEANS

Comment (chin2yere) [reply Petits-debuts-ui/src/Components/Trending/Trending.jsx]: okay, thanks

**Pull request merged**